apps/web/src/components/HintsAccordion/HintsAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Heading,
+} from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { type SignPage } from "../Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+
+export const Hints: Record<SignPage, HintData> = {
+  TezSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  ContractCallSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  DelegationSignPage: {
+    header: "It takes 4 cycles (~6 days) to start receiving delegation rewards.",
+    description:
+      "Delegation rewards are distributed every 2 cycles (~3 days). Ensure that the delegation fee offered by the baker is less than 100%; otherwise, you will not receive rewards. Delegation means that all your funds are delegated to one baker. The funds are still spendable, and you can cancel or change the delegation at any time.",
```

Comment:
```suggestion
      "Bakers usually distribute delegation rewards  every cycle (~2.8 days). Ensure that the delegation fee offered by the baker is less than 100%; otherwise, you might not receive rewards. Delegation means that all your funds are delegated to one baker. The funds are still spendable, and you can cancel or change the delegation at any time.",
```
Delegation rewards are allocated to the baker during the cycle, they usually pay them a number of cycles in delay. This is not mandated by the protocol. Bakers might pool rewards and pay them each k cycles, for instance